tests/conftest.py

```diff
@@ -1,70 +1,147 @@
 from pathlib import Path
+from tempfile import mkdtemp
 
-import pytest  # type: ignore
-from ape import Project, accounts, networks
-from ape.api.networks import LOCAL_NETWORK_NAME, NetworkAPI
-from ape.managers.project import ProjectManager
+import ape
+import pytest
+from _pytest.runner import pytest_runtest_makereport as orig_pytest_runtest_makereport
+from ape.api.networks import LOCAL_NETWORK_NAME
+from ape.contracts import ContractContainer, ContractInstance
+from ape.logging import LogLevel, logger
+from ethpm_types import ContractType
 
-from ape_foundry import FoundryProvider
+from ape_foundry import FoundryForkProvider, FoundryProvider
 
+# NOTE: Ensure that we don't use local paths for the DATA FOLDER
+ape.config.DATA_FOLDER = Path(mkdtemp()).resolve()
 
-def get_project() -> ProjectManager:
-    return Project(Path(__file__).parent)
+BASE_CONTRACTS_PATH = Path(__file__).parent / "data" / "contracts"
+logger.set_level(LogLevel.DEBUG)
```

Comment:
can get rid of this once https://github.com/ApeWorX/ape/pull/889 merges and is released and usable